src/index.spec.js

```diff
@@ -159,10 +159,12 @@ describe('provideState', () => {
 
   describe('computed', () => {
     const sum = jest.fn(({ foo }, { bar }) => foo + bar)
+    const circularComputed = jest.fn(({ circularComputed }, { bar }) => circularComputed + bar)
```

Comment:
No need to use a prop, and you can leave the body empty, the state destructuring is enough :slightly_smiling_face: